llvm/include/llvm/Support/SpecialCaseList.h

```diff
@@ -132,14 +132,15 @@ class SpecialCaseList {
   using SectionEntries = StringMap<StringMap<Matcher>>;
 
   struct Section {
-    Section(std::unique_ptr<Matcher> M) : SectionMatcher(std::move(M)){};
-    Section() : Section(std::make_unique<Matcher>()) {}
+    Section(std::unique_ptr<Matcher> M) : SectionMatcher(std::move(M)) {};
```

Comment:
Try  to avoid mixing formatting unrelated changes with logic changes.
Either revert or move into a new PR